ihatemoney/templates/list_bills.html

```diff
@@ -128,6 +128,9 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
                 <td class="bill-actions">
                     <a class="edit" href="{{ url_for(".edit_bill", bill_id=bill.id) }}" title="{{ _("edit") }}">{{ _('edit') }}</a>
                     <a class="delete" href="{{ url_for(".delete_bill", bill_id=bill.id) }}" title="{{ _("delete") }}">{{ _('delete') }}</a>
+                    {% if bill.document %}
+                    <a class="see" href="{{ bill.document }}" target="_blank" title="{{ _("see") }}">{{ _('see') }} </a>
```

Comment:
i would add `ref="noopener"` and maybe noreferrer ... but the latest is more questionable...